docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: hexpm_dev
```

Comment:
We don't need this since `mix ecto.create` creates the database, correct?

Reply:
Correct